lazyflow/utility/io_util/OMEZarrStore.py

```diff
@@ -0,0 +1,180 @@
+###############################################################################
+#   ilastik: interactive learning and segmentation toolkit
+#
+#       Copyright (C) 2011-2024, the ilastik developers
+#                                <team@ilastik.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# In addition, as a special exception, the copyright holders of
+# ilastik give you permission to combine ilastik with applets,
+# workflows and plugins which are not covered under the GNU
+# General Public License.
+#
+# See the LICENSE file for details. License information is also available
+# on the ilastik web site at:
+# 		   http://ilastik.org/license.html
+###############################################################################
+import json
+import logging
+import os
+from collections import OrderedDict
+from typing import Dict
+from urllib.parse import unquote_to_bytes
+
+import jsonschema
+import vigra
+from zarr.core import Array as ZarrArray
+from zarr.storage import FSStore, LRUStoreCache
+
+from lazyflow import rtype
+from lazyflow.utility import Timer
+from lazyflow.utility.io_util.multiscaleStore import MultiscaleStore, DEFAULT_SCALE_KEY
+
+logger = logging.getLogger(__name__)
+
+OME_ZARR_V_0_4_KWARGS = dict(dimension_separator="/", normalize_keys=False)
+OME_ZARR_V_0_1_KWARGS = dict(dimension_separator=".")
+
+
+def _get_axistags_from_spec(ome_spec: Dict) -> vigra.AxisTags:
+    if "axes" not in ome_spec:
+        # v0.1 and v0.2 did not allow variable axes
+        axis_keys = ["t", "c", "z", "y", "x"]
+    else:
+        ome_axes = ome_spec["axes"]
+        assert isinstance(ome_axes, list), f"Expected axis information to be a list, received: {ome_axes}."
```

Comment:
Assertions are really there to pick up on bugs in our application, but here you're checking for a condition on a file that could have been malformed. We should foresee that we could encounter malformed files in the wild, and react appropriately to them.

Reply:
The input here is already `jsonschema.validate`d; I guess it would be more consistent to just remove the assert altogether.